Detector/DetSensitive/DetSensitive/SimpleCalorimeterSDWithBirksLaw.h

```diff
@@ -0,0 +1,62 @@
+#ifndef DETSENSITIVE_SIMPLECALORIMETERSD_WITHBIRKS_H
+#define DETSENSITIVE_SIMPLECALORIMETERSD_WITHBIRKS_H
+
+// DD4hep
+#include "DDG4/Geant4Hits.h"
+#include "DDSegmentation/Segmentation.h"
+
+// Geant
+#include "G4VSensitiveDetector.hh"
+#include "G4THitsCollection.hh"
+
+/** SimpleCalorimeterSD DetectorDescription/DetSensitive/src/SimpleCalorimeterSD.h SimpleCalorimeterSD.h
+ *
+ *  Simple sensitive detector for calorimeter.
+ *  It is based on DD4hep::Simulation::Geant4GenericSD<Calorimeter> (but it is not identical).
+ *  In particular, the position of the hit is set to G4Step::GetPreStepPoint() position.
+ *  New hit is created for each energy deposit.
+ *  No timing information is saved.
+ *  Birks law is included for polystyrene
+ *  @author    Coralie Neubueser
+ */
+
+namespace det {
+class SimpleCalorimeterSDWithBirksLaw : public G4VSensitiveDetector {
```

Comment:
Since this implementation is quite far from being simple (and also because it is a very long name), can you change the name of this class to `BirksLawCalorimeterSD` or something similar?